src/theme/ui_styles.rs

```diff
@@ -31,12 +31,24 @@ pub struct UiStyles {
     pub broken_path_overlay:  Style,  // bO
 }
 
+#[derive(Clone, Copy, Debug, PartialEq)]
+pub enum LinkStyle {
+    AnsiStyle(Style),
+    Target,
+}
+
+impl Default for LinkStyle {
```

Comment:
Nit: can this be derived?

Reply:
Sorry. I don't know what that means. I don't actually know Rust. I just took the original patch and tweaked it to work with the current code

If you can point me in the right direction though, I'm happy to take a look